Retsinformation.js

```diff
@@ -0,0 +1,191 @@
+		newItem.docketNumber = getKortNavn(doc, url).id;
+		newItem.dateDecided = getKortNavn(doc, url).date;
+		newItem.shortTitle = getRessort(doc, url).shortTitle;
+	} else if (type=="report") {
+		newItem = addAuthor (newItem, getRessort(doc, url).ressort);
+		newItem.date = getRessort(doc, url).pubDate;
+		newItem.institution = "Folketinget";
+		threeLetters = getKortNavn(doc, url).threeLetters;
+		if (threeLetters=="EDP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands driftundersøgelser";
+		} else if (threeLetters=="ISP") {
+			newItem.seriesTitle = "Folketingets Ombudsmands Inspektioner";
+		} else if (threeLetters=="FOU") {
+			newItem.seriesTitle = "Folketingets Ombudsmands udtalelser";
+		};
+		newItem.reportNumber = getKortNavn(doc, url).id;
+	} else if (type=="bill") {
+		newItem = addAuthor (newItem, getRessort(doc, url).ressort);
+		newItem.billNumber = getKortNavn(doc, url).id;
+		newItem.date = getRessort(doc, url).pubDate;
+	}
+	newItem.complete();
+}
+
+function getType (doc, url) {
+	threeLetters = getKortNavn(doc, url).threeLetters;
+	if (/(DOM|AFG|KEN|UDT)/.test(threeLetters)) {
+		return "case";
+	}
+	if (/(EDP|ISP|FOU)/.test(threeLetters)) {
+		return "report";
+	}
+	if (/\d{3}/.test(threeLetters)) {
+		return "bill";
+	} else {
+		return "statute";
+	}
+}
+
+function getTitle (doc, url) {
+	var baseXPath = '//div[@class="wrapper2"]/div/';
+	var xPath1 = baseXPath + 'p[@class="Titel2"]';
+	baseXPath = baseXPath + 'div[@id="INDHOLD"]/';
+	var xPath2 = baseXPath + 'p[@class="Titel"]';
+	var xPath3 = baseXPath + 'font/p[@align="CENTER"]';
+	var xPath4 = baseXPath + 'h1[@class="TITLE"]';
+	var xPath5 = baseXPath + 'center/h1[@class="TITLE"]';
+	var title = ZU.xpathText(doc, xPath1) || ZU.xpathText(doc, xPath2) || ZU.xpathText(doc, xPath3) || ZU.xpathText(doc, xPath4) || ZU.xpathText(doc, xPath5);
+	if (!title) {
+		var myXPath = '/html/head/title';
+		title = ZU.xpathText(doc, myXPath);
+		title = title.substring(0,title.lastIndexOf("-"));
+		if (getRessort(doc, url)[0]) {
+			title = title.substring(getRessort(doc, url)[0].length+4,title.length);
+		};
+	}; 
+	title = ZU.trimInternal(title);
+	return title;
+}
+
+function getKortNavn (doc, url) {
+	var myXPath = '//span[starts-with(@class,"kortNavn")]';
+	var fodder = ZU.xpathText(doc, myXPath);
+	fodder = fodder.replace(/ (Gældende|Historisk)/,"");
+	var item = new Object();
+	item.threeLetters = fodder.substr(0,3);
+	fodder = fodder.split(" af ");
+	item.date = fodder[1];
+	item.id = fodder[0];
+	return item;
+}
+
+function getRessort (doc, url) {
+	var myXPath = '//div[@class="ressort"]';
+	var fodder = ZU.xpathText(doc, myXPath);
+	fodder = fodder.trim();
+	var item = new Object();
+	if (fodder.charAt(0) == "(") {
+		item.shortTitle = fodder.substring(1,fodder.lastIndexOf(")")-1);
+	}
+	var datePosition = fodder.search(/\d{2}\-\d{2}\-\d{4}/);
+	item.ressort = fodder.substr(datePosition+10);
+	item.pubDate = fodder.substr(datePosition,10);
```

Comment:
I've added a note to the code here.
I could get the date using 'match' instead of 'search', but I can't see how I can extract the 'ressort' part of the string - it's the last part of the string after the date.